src/common/components/feedback/index.tsx

```diff
@@ -89,74 +143,131 @@ export default class Feedback extends BaseComponent<Props, State> {
+            switch (x.type) {
+              case "success":
+                return (
+                  <div key={x.id} className="feedback-success">
+                    <div className="feedback-body">
+                      <div
+                        className="feedback-close-btn"
+                        onClick={() =>
+                          this.setState({ display: false, progress: 100, intervalState: true })
+                        }
+                      >
+                        {closeSvg}
+                      </div>
+                      <div className="feedback-content">
+                        <div className="feedback-img success-img">{checkSvg}</div>
+                        {x.message}
+                      </div>
+                    </div>
+
+                    <div className="toast-progress-bar">
+                      <div className="filler success" style={style} />
+                    </div>
+                  </div>
+                );
+              case "error":
+                return (
+                  <div key={x.id} className="feedback-error align-items-start">
+                    <div className="feedback-body">
+                      <div
+                        className="feedback-close-btn"
+                        onClick={() =>
+                          this.setState({ display: false, progress: 100, intervalState: true })
+                        }
+                      >
+                        {closeSvg}
+                      </div>
+                      <div className="error-content">
+                        <div className="error-img">{alertCircleSvg}</div>
+
+                        <div className=" d-flex flex-column align-items-start">
+                          {x.message}
+                          <div className="d-flex">
+                            {errorType(x) !== ErrorTypes.COMMON &&
+                            errorType(x) !== ErrorTypes.INFO ? (
+                              <Button
+                                className="mt-2 details-button px-0 mr-3"
+                                variant="link"
+                                onClick={() =>
+                                  this.setState({ showDialog: true, detailedObject: x })
+                                }
+                              >
+                                {_t("feedback-modal.question")}
+                              </Button>
+                            ) : (
+                              <></>
+                            )}
+                            {!ErrorTypes.INFO && (
+                              <Button
+                                className="mt-2 details-button px-0"
+                                variant="link"
+                                onClick={() =>
+                                  window.open(
+                                    "mailto:bug@ecency.com?Subject=Reporting issue&Body=Hello team, \n I would like to report issue: \n",
```

Comment:
I guess these sentences should be translated

Reply:
https://github.com/ecency/ecency-vision/pull/1196#discussion_r1145387045